pkg/registry/serving/composer.go

```diff
@@ -191,13 +191,21 @@ func getUserContainerImage(pu *seldonv1.PredictiveUnit) string {
 func getRuntimeResource(pu *seldonv1.PredictiveUnit) (*corev1.ResourceRequirements, error) {
 	cpu := ""
 	mem := ""
+	gpuType := ""
+	gpuNum := ""
 	for _, p := range pu.Parameters {
 		if p.Name == "cpu" {
 			cpu = p.Value
 		}
 		if p.Name == "mem" {
 			mem = p.Value
 		}
+		if p.Name == "gpuType" {
```

Comment:
Is it built-in in seldon core?

Reply:
YES, It can be set in  `Container.Resource Field` directly, but I think seldon core is too complex（because seldondeployment CRD has two tree structure - graph tree and component tree，and it must match by name），so，all infos pass by Graph struct，the component‘s info is set by backend.

Reply:
Then how to support CPU like 10m, 1000m, 1, and Mem like 100Mi, 100M if we do not use resource.Quantity?

Reply:
It is `resource.ParseQuantity` already( line 256 ~ line 231),  the params's typ is string, it is parsed by`resource.ParseQuantity(gpuNum)`.